defaults/main.yml

```diff
@@ -34,3 +34,11 @@ haproxy_enable_syslog: false
 haproxy_use_graylog_json_logging: false
 haproxy_graylog_server:
 haproxy_graylog_server_port:
+
+# Prometheus exporter support for haproxy v1.8
+haproxy_external_exporter_enabled: false
+haproxy_exporter_download_url: https://github.com/prometheus/haproxy_exporter/releases/download/v0.12.0/haproxy_exporter-0.12.0.linux-amd64.tar.gz
+haproxy_exporter_listen_port: 9101
+haproxy_exporter_stats_user: admin
+haproxy_exporter_stats_password: password
```

Comment:
Do I even wanna know why there's a default password of `password` here?

Reply:
No